app/grants/models.py

```diff
@@ -2307,3 +2307,141 @@ class GrantBrandingRoutingPolicy(SuperModel):
 
     def __str__(self):
         return f'{self.url_pattern} >> {self.priority}'
+
+
+class GrantHallOfFame(SuperModel):
```

Comment:
Im not sure the way was implemented actual fit the system needs. 
Let me explain better this: 
On the design I see the use of profile users data and grants data, but here on the model all new fields are being created instead of foreing keys to the actual data, that make the admin need to reupload for example the user avatar and if that get changed the admin will need to figure out and come back to change it here also. 
In general I understand this fit the design but doesn't fit the data base relation on the platform. 
Those relation are very important because make it possible to extend the system to new features, lets say in the future we want t show a badge on the profile based on a user that was on a hall of fame, or want to show a signal on the grants of those users, or want to filter by the grants on the hall of fame, as there is no relation with the actual data this make it a total isolated page without possible extension.

Reply:
Hello @octavioamu 

Yes, what you say about reusing the user profile data, makes total sense. This however will only affect the "Graduated Grantees" section. I assume I can reuse the logo, username and name .. but I will need to check the oder grant information. I will proceed and make these changes.

Regarding the second part of your comment, the data model for the hall of fame was not modelled this way. It was only created as a way for the marketing team for example to update the "Hall of Fame" page - this is how I understood the requirement. So as it currently is, it is not intended to track if a user was in the hall of fame or not or track the grants.
@octavioamu Is this part of the requirement? Did I misunderstand it?

Reply:
I have now linked the Profiles to the "Graduated Grantees" list. The handle and avatar are taken from the profile.